### ISSUE TITLE
[RLlib] Inference does not work when action space is spaces.Tuple

### ISSUE THREAD

**Issue by awagmanno** [ray-project/ray#39761]
### What happened + What you expected to happen

I have environment that uses spaces.Tuple for the action space of the environment. I am using PPO. Training works fine but when I try to compute an action with "algo.compute_single_action(obs, explore = False)", so, setting the explore variable to False it does not work and I get the following error.

`/venv/lib/python3.10/site-packages/ray/rllib/models/torch/torch_distributions.py", line 494, in <lambda>
    return tree.map_structure(lambda s: s.sample(), child_distributions_struct)`

When the explore variable is set to true it works, but in that case the actions are not deterministic which is what I need.

### Versions / Dependencies

Python version: 3.10
Ray Version: 2.6.2
Gymnasium Version: 0.28.1
OS Version: Pop!_OS 22.04 LTS

### Reproduction script

```python
from gymnasium import Env, spaces
import numpy as np
from ray.rllib.algorithms.ppo import PPOConfig

class TestEnv(Env):

    def __init__(self, config = {}):
        self.observation_space = spaces.Box(low = -2**63, high = 2**63, shape = (1,), dtype = np.float64)
        self.action_space_1 = spaces.Box(-1.0, 1.0, shape = (1,), dtype = np.float64)
        self.action_space_2 = spaces.Discrete(2)
        self.action_space = spaces.Tuple((self.action_space_1, self.action_space_2))
        # self.action_space = self.action_space_2

    def reset(self, seed = None, options = None):
        self.steps = 0
        self.state = 0
        return np.array([1.0]), {}

    def step(self, action):
        on_off = action[1]
        value = float(action[0])

        self.steps += 1
        self.state += value * on_off

        if self.state < -10:
            self.state = -10
        
        if self.state > 10:
            self.state = 10
        truncate = self.truncate()
        return np.array([self.state]), self.state/10, False, truncate, {}
    
    def close(self):
        pass
    
    def truncate(self):
        return False if self.steps < 500 else True
    
config = PPOConfig()
config.environment(env=TestEnv)

algo = config.build()

terminated = truncated = False
env = TestEnv()
obs, info = env.reset()
while not terminated and not truncated:
# The error is here
    action = algo.compute_single_action(obs, explore = False)

    obs, reward, terminated, truncated, info = env.step(action)






```

### Issue Severity

Medium: It is a significant difficulty but I can work around it.

**Comment by awagmanno** [comment]
Problem fixed in #39411

**Comment by awagmanno** [comment]
Problem fixed in #39411
